### PR TITLE
Added few Channels

### LINKED ISSUE #4139: Sun News
**Channel Name:** Sun News
**Country:** IN
**Language:** Tamil
**Website:** http://www.sunnewslive.in/
**Stream URL (optional):** https://www.youtube.com/watch?v=xnk9g38W_5w
**Notes (optional):** xxx

### LINKED ISSUE #4145: Malabar News
**Channel Name:** Malabar News 
**Country:** IN
**Language:** Malayalam
**Website:** http://malabarnewschannel.com
**Stream URL (optional):** http://malabarnewschannel.com/newslive.html
**Notes (optional):** xxx

### LINKED ISSUE #4167: Dreamz
**Channel Name:** Dreamz
**Country:** India
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** http://expressott.in/6d2dc651-d89f-469d-b46b-6329a0e025b4
**Notes (optional):** xxx

### LINKED ISSUE #4170: M Sign Media
**Channel Name:** M Sign Media
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** http://expressott.in/7f7aa3f9-e9cb-4fcf-8de7-1416bc3c5da1
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by Niyasmkkv**
- Adds Dreamz TV ( Closes #4167 )
- Adds Malabar News ( Closes #4145 )
- Adds M Sign Media ( Closes #4170 )
- Adds Sun News ( Closes #4139 )
- Fixes Rosebowl & ADN Gold
- Ditches some specific quality streams for their adaptive playlist.

### CODE REVIEW

**Comment by Niyasmkkv** [comment]
@Dum4G  what went wrong? I'm clueless.

**Comment by LaneSh4d0w** [comment]
Few trailing spaces on the channel names.

File: channels/in.m3u [hunk 1]
```diff
@@ -375,6 +381,8 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 https://live.mycast.in/encode/ee0c5a36ff5a7083ee044991974ad3ba.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="PratidinTime.in" tvg-name="Pratidin Time" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="News",Pratidin Time (360p)
 https://rtmp.smartstream.video/pratidintime/pratidintime/playlist.m3u8
+#EXTINF:-1 tvg-id="PravasiTV.in" tvg-name="Pravasi TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="News",Pravasi TV
+https://5ee50688d7b5d.streamlock.net:444/live/pravasi/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:385]: Doesn't seem to work for me.

Comment (Niyasmkkv) [reply channels/in.m3u:385]: It was working fine. Not working now. can you remove it?

Comment (LaneSh4d0w) [reply channels/in.m3u:385]: I'll wait and try again tomorrow, might simply be Not 24/7.

Comment (Niyasmkkv) [reply channels/in.m3u:385]: Not playing in the source website also. Error Code: 232404)

Comment (LaneSh4d0w) [reply channels/in.m3u:385]: Still not working right now. Deleting it.

**Comment by LaneSh4d0w** [comment]
Took advantage of the fact we were working on the Indian playlist to make some additional modifications.

**Review by Dum4G — APPROVED** [review]

**Comment by Dum4G** [comment]
Since I'm away from the Internet I'll leave a pre-approval for this PR

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**